src/s2wasm.h

```diff
@@ -961,7 +961,11 @@ class S2WasmBuilder {
       Const* curr = triple.value;
       Name name = triple.name;
       size_t offset = triple.offset;
-      curr->value = Literal(staticAddresses[name] + offset);
+      const auto &symbolAddress = staticAddresses.find(name);
+      if (symbolAddress == staticAddresses.end()) {
+        abort_on("Unknown symbol");
```

Comment:
Done. I was thinking about reworking the abort_on stuff at some point, it's not super friendly. Tracking line + column would be more helpful I think.

Maybe later :)